packages/react/src/components/IonPickerLegacy.tsx

```diff
@@ -1,9 +1,9 @@
 import type { JSX } from '@ionic/core/components';
-import { defineCustomElement } from '@ionic/core/components/ion-picker.js';
+import { defineCustomElement } from '@ionic/core/components/ion-picker-legacy.js';
 
 import { createInlineOverlayComponent } from './createInlineOverlayComponent';
 
-export const IonPicker = /*@__PURE__*/ createInlineOverlayComponent<JSX.IonPicker, HTMLIonPickerElement>(
-  'ion-picker',
-  defineCustomElement
-);
+export const IonPickerLegacy = /*@__PURE__*/ createInlineOverlayComponent<
+  JSX.IonPickerLegacy,
+  HTMLIonPickerLegacyElement
+>('ion-picker', defineCustomElement);
```

Comment:
This is the tag of the component, right? So this should have changed?

Reply:
Yes, this is what developers import when they want to use the picker.

Reply:
Sorry for the confusion. I'm inquiring about line 9.

https://github.com/ionic-team/ionic-framework/blob/cf7882172a13d9253d6d49e4becfcf5641df68df/packages/react/src/components/IonPickerLegacy.tsx#L9

Reply:
Oh thanks! Yes, that should say `ion-picker-legacy`. Fixed in https://github.com/ionic-team/ionic-framework/pull/28584/commits/5180cb4c2c16861ee6bc6d83ddd200d26eacc405